roles/KubevirtNodeLabeller/templates/kubevirt-node-labeller-ds.yaml.j2

```diff
@@ -44,7 +44,7 @@ spec:
               name: cpu-config
 
         - name: libvirt
-          image: {{ ssp_registry | default('kubevirt') }}/{{ libvirt_image }}:{{ libvirt_tag }}
+          image: {{ lookup('env','VIRT_LAUNCHER_IMAGE')| default(virt_launcher_image, true) }}
```

Comment:
please elaborate about what this line is supposed to do should VIRT_LAUNCHER_IMAGE not be defined

Reply:
in retrospect I think it would be better to use two env vars, one for the libvirt_image and one for the libvirt_tag. I think it should look like

 image: {{ ssp_registry | default('kubevirt') }}/{{ lookup('env','VIRT_LAUNCHER_IMAGE')| default(libvirt_image, true) }}:{{ lookup('env','VIRT_LAUNCHER_TAG')| default(libvirt_tag, true) }}